src/vos/vos_aggregate.c

```diff
@@ -1609,7 +1609,6 @@ vos_aggregate(daos_handle_t coh, daos_epoch_range_t *epr)
 	agg_param.ap_discard = false;
 	merge_window_init(&agg_param.ap_window);
 
-	iter_param.ip_flags |= VOS_IT_FOR_PURGE;
 	rc = vos_iterate(&iter_param, VOS_ITER_OBJ, true, &anchors,
```

Comment:
I thought the idea behind PURGE was for deleting orphaned records (e.g. for txns that started but then client dies or some such so they are stuck forever).   Probably better to handle that another way but that was my understanding for why aggregation would want to see them.

Reply:
I guess the right thing to do would be to return them but marked covered?

Reply:
Right, I think in the future, we need to rely on caller to ensure that all the records in the aggregated epr are either committed or to be aborted, then we can safely delete the uncommitted one during aggregation.